src/components/common/Pagination/Pagination.test.tsx

```diff
@@ -0,0 +1,93 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+
+import "@testing-library/jest-dom";
+
+import { describe, expect, it, vi } from "vitest";
```

Comment:
Take out this line, it is available globally

Reply:
I tried to check how it was done in other code but it seems nothing much was changed

Any pointer on that will be appreciated